test/spec/modules/nextrollBidAdapter_spec.js

```diff
@@ -11,7 +11,7 @@ describe('nextrollBidAdapter', function() {
 
   afterEach(function() {
     global.NextRoll = undefined;
-    utilsMock.restore();
+    utilsMock?.restore();
```

Comment:
were you able to verify that this really helps? logically I don't think it can. you are seeing the error here because something broke `beforeEach`, even if you "fix" this every test in this suite will still fail without its setup.